frontend/src/v5/ui/v4Adapter/overrides/colorPicker.overrides.ts

```diff
@@ -39,4 +39,41 @@ export default css`
 			}
 		}
 	}
+	
+	.color-picker__panel { // Colour picker modal
```

Comment:
It's strange how this comment isn't causing a linter error

Reply:
Yeah, I don't get why we get errors in the first place sometimes when we use comments